static/js/util/profile_edit.js

```diff
@@ -71,7 +73,68 @@ export function boundDateField(key, label, error) {
     <span className="validation-error-text">{error}</span>
   </div>;
 }
+export function boundEducationMonthField(keySet, label) {
+  const {updateProfile, profile} = this.props;
+  let onChange = month => {
+    let clone = Object.assign({}, profile);
 
+    let current_value = _.get(clone, keySet);
+        // format as ISO-8601
+    if(month === null){
```

Comment:
Can you change this if statement to set a local variable instead, and use `_.set` once at the end of the if statement with the local variable?